surface/secretsmanager/management/commands/import_git_secrets.py

```diff
@@ -0,0 +1,217 @@
+                )
+
+                files = []
+                for line in files_result.stdout.splitlines():
+                    try:
+                        mode, type, hash, path = line.split(None, 3)
+                        files.append(path)
+                    except ValueError:
+                        continue
+
+                self.stdout.write(f"Files in commit {commit[:8]}: {files}")
+
+                sensitive_files = [
+                    f for f in files 
+                    if any(f.lower().endswith(ext.lower()) for ext in allowed_formats)
+                ]
+                
+                if sensitive_files:
+                    self.stdout.write(
+                        self.style.SUCCESS(
+                            f"Found sensitive files in {commit[:8]}: {sensitive_files}"
+                        )
+                    )
+                    
+                    for file_path in sensitive_files:
+                        author = subprocess.run(
+                            f"git -C {repo_path} log -1 --format=%ae {commit} -- {file_path}",
+                            stdout=subprocess.PIPE,
+                            shell=True,
+                            text=True
+                        ).stdout.strip()
+                        
+                        date = subprocess.run(
+                            f"git -C {repo_path} log -1 --format=%aI {commit} -- {file_path}",
+                            stdout=subprocess.PIPE,
+                            shell=True,
+                            text=True
+                        ).stdout.strip()
+
+                        yield {
+                            'commit': commit,
+                            'file': file_path,
+                            'author': author,
+                            'date': date
+                        }
+
+        except subprocess.CalledProcessError as e:
+            self.stdout.write(self.style.ERROR(f"Error scanning git history: {e}"))
+
+    def calculate_file_hash(self, repo_path, commit_hash, file_path):
+        """Calculate SHA256 hash of file content at specific commit"""
+        try:
+            result = subprocess.run(
+                f"git -C {repo_path} show {commit_hash}:{file_path}",
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                shell=True,
+                check=True
+            )
+            content = result.stdout
+            return hashlib.sha256(content).hexdigest()
+        except subprocess.CalledProcessError as e:
+            self.stdout.write(self.style.ERROR(f"Error reading file {file_path}: {e}"))
+            return None
+
+    def handle(self, *args, **options):
+        repo_path = options['repo_path']
+        self.org = options['org']
+
+        self.stdout.write(f"Scanning repository: {repo_path}")
+        
+        repo_name = os.path.basename(repo_path)
+        repo_url = f"https://github.com/{self.org}/{repo_name}"
```

Comment:
We have to think well about this. As it stands, seems it would only work for `github.com` and it's important to support other platforms here, in my perspective (don't know about others).